modules/react-components/src/components/message/message-info.tsx

```diff
@@ -34,6 +34,10 @@ export interface MessageInfoProps extends TestableComponentInterface {
      * Shorthand for primary content.
      * */
     content?: SemanticShorthandContent;
+    /**
+     * Callback method for handle close icon.
+     * */
+    handleCloseInfo?: () => void;
```

Comment:
Shall we extend `MessageInfoProps` with `MessageProps`(from semantic)?
Then you can use the original props from semantic message component.